Source/WebCore/editing/FrameSelection.cpp

```diff
@@ -431,23 +432,23 @@ bool FrameSelection::setSelectionWithoutUpdatingAppearance(const VisibleSelectio
     setCaretRectNeedsUpdate();
 
     if (!newSelection.isNone() && !(options & SetSelectionOption::DoNotSetFocus)) {
-        auto* oldFocusedElement = m_document->focusedElement();
+        auto* oldFocusedElement = document->focusedElement();
```

Comment:
Use RefPtr here as well?

Reply:
It's not being Dereferenced so I wasn't sure but OK.